src/theme/components/socialmedialinks.ts

```diff
@@ -0,0 +1,208 @@
+// @todo maybe set the border to always be on, but invisible if prop is set to "none"?
+
+import { StyleFunctionProps } from "@chakra-ui/theme-tools";
+
+function getPadding(size) {
+  let paddingSize = "xs"; // 8px / .5rem
+  switch (size) {
+    case "medium":
+      paddingSize = "0.75rem"; // 12px @todo There is no JS token for this value. I assume it is ok to mix tokens & absolute values.
+      break;
+    case "large":
+      paddingSize = "s"; // 16px / 1rem
+      break;
+  }
+  return {
+    paddingSize,
+  };
+}
+
+function getColor(color) {
+  let ltColor = "ui.typography.heading",
+    dkColor = "dark.ui.typography.heading";
+
+  switch (color) {
+    case "link":
+      ltColor = "ui.link.primary";
+      dkColor = "dark.ui.link.primary";
+      break;
+    case "textInverse":
+      ltColor = "ui.typography.inverse.heading";
+      dkColor = "dark.ui.typography.inverse.heading";
+      break;
+  }
+
+  return {
+    ltColor,
+    dkColor,
+  };
+}
+
+const SocialMediaLinks = {
+  // The base styles for each part
+  baseStyle: {
+    // ul styles should not specifically identified as ul:
+    display: "flex",
+    flexDirection: { base: "column", md: "row" }, // @todo Storybook doesn't seem to dynamically update this when you change the viewport control.
```

Comment:
Are you manually changing viewports? Storybook also has a viewport helper in the top nav:

<img width="446" alt="Screen Shot 2023-09-07 at 3 59 20 PM" src="https://github.com/NYPL/nypl-design-system/assets/1280564/cd60ac78-9022-4da4-971b-4d0154016541">